src/runtime-tools/linux/run.sh

```diff
@@ -19,6 +19,7 @@ echo core | sudo tee /proc/sys/kernel/core_pattern
 echo 0 | sudo tee /proc/sys/kernel/randomize_va_space
 # set core dumping to default behavior
 echo 1 | sudo tee /proc/sys/fs/suid_dumpable
+RUNID=$(uuidgen)
```

Comment:
I double-checked, and confirmed that this is also installed on a stock Ubuntu 20.04 image. ✔️